src/Symfony/Component/Console/Application.php

```diff
@@ -1082,7 +1088,8 @@ protected function getDefaultInputDefinition(): InputDefinition
         return new InputDefinition([
             new InputArgument('command', InputArgument::REQUIRED, 'The command to execute'),
             new InputOption('--help', '-h', InputOption::VALUE_NONE, 'Display help for the given command. When no command is given display help for the <info>'.$this->defaultCommand.'</info> command'),
-            new InputOption('--quiet', '-q', InputOption::VALUE_NONE, 'Do not output any message'),
+            new InputOption('--silent', null, InputOption::VALUE_NONE, 'Do not output any message'),
+            new InputOption('--quiet', '-q', InputOption::VALUE_NONE, 'Only errors are displayed. All other output is suppressed'),
```

Comment:
```suggestion
            new InputOption('--quiet', '-q', InputOption::VALUE_NONE, 'Suppress all output, except errors'),
```
?

Reply:
I've copy pasted the original from the AWS CLI, I have no strong opinion between the two.

Unless someone has a strong opinion, I would prefer not to have to update it 100 times in the test suite again :wink:

Reply:
Fine then